.github/workflows/close-stale-enhancement-requests.yaml

```diff
@@ -14,9 +14,9 @@ jobs:
     steps:
       - uses: actions/stale@v9
         with:
-          debug-only: true # Dry-run
+          debug-only: true # Uncomment to switch back to dry-run
```

Comment:
I intend to leave this line in just commented out once we enable this. I'm leaving this so we don't have to go look at docs in case we want to switch to dry-run quickly. 

I recognize that we can just disable the workflow if needed.